app/helpers/statuses_helper.rb

```diff
@@ -130,6 +130,7 @@ def render_video_component(status, **options)
       blurhash: video.blurhash,
       frameRate: meta.dig('original', 'frame_rate'),
       inline: true,
+      autoFocus: embedded_view? ? false : options.detail,
```

Comment:
I don't think you need to do that, I think you can just drop it! Tying the autofocus to detailed view was a mistake from my part imho.